modules/management-console-agent/src/main/java/org/apache/ignite/agent/action/controller/SecurityActionsController.java

```diff
@@ -48,7 +48,7 @@
     public SecurityActionsController(GridKernalContext ctx) {
         this.ctx = ctx;
         this.log = ctx.log(SecurityActionsController.class);
-        this.registry = ((ManagementConsoleProcessor)ctx.managementConsole()).sessionRegistry();
+        this.registry = ((ManagementConsoleAgent)ctx.managementConsole()).sessionRegistry();
```

Comment:
Why we need this cast. Can we add this method to interface?

Reply:
No, this is implementation details so far.